client/src/components/RequestModal.js

```diff
@@ -56,25 +57,29 @@ export default class RequestModal extends Component {
         }
         this.handleValueChange = this.handleValueChange.bind(this)
         this.submitRequest = this.submitRequest.bind(this)
+        this.handleOffsetChange = this.handleOffsetChange.bind(this)
+    }
+    
+    async handleOffsetChange(offset) {
+        if (this.state.timeOffset !== offset) {
+            window.location.reload()
```

Comment:
This is gross - hopefully there is a cleaner way of doing this, but using refresh profile wasn't quick enough to fetch the profile in time for alumni, which caused an error